app/static/openapi.yaml

```diff
@@ -560,51 +572,53 @@ paths:
               example:
                 apiVersion: '1.0'
                 data: null
+                error: 1
                 errors:
                   unprocessable-entity:
                     message: 'The value for "updated_after" is invalid'
                 status: 'Unprocessable Entity'
                 status_code: 422
 
     post:
-      summary: Create Resource
-      description: Creates a new learning resource entry. The required properties for a new entry are `category`, `name`, `paid`, and `url` and must be included in the request body.
+      summary: Create Resources
+      description: Creates new learning resources. The structure of the request is a list of resource objects. The required properties for each new resource are `category`, `name`, `paid`, and `url`.
       tags:
-        - Create Resource
+        - Create Resources
       requestBody:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/CreateResourceRequest'
+              $ref: '#/components/schemas/CreateResourcesRequest'
             example:
-              category: 'Tutorial'
-              name: 'CSS Tutorial'
-              paid: false
-              url: 'https://www.w3schools.com/css/'
+              - category: 'Tutorial'
+                name: 'CSS Tutorial'
+                paid: false
+                url: 'https://www.w3schools.com/css/'
         required: true
       responses:
         200:
           description: Created resource
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Resource'
+                $ref: '#/components/schemas/Resource_list'
```

Comment:
why is this underscore delimited instead of camel-case?

Reply:
I was following the existing pattern of `Something_list` that was in there. This file could probably use a little name convention refactoring. I'm not really sure what best practices are